tools/mk/Makefile.targ

```diff
@@ -198,7 +159,7 @@ xref: cscope.files
 .PHONY: cscope.files
 cscope.files:
 	find $(CSCOPE_DIRS) -name '*.c' -o -name '*.h' -o -name '*.cc' \
-	    -o -name '*.js' -o -name '*.s' -o -name '*.cpp' > $@
+		-o -name '*.js' -o -name '*.s' -o -name '*.cpp' > $@
```

Comment:
Not sure what happened here, but normalized all the tabs in the files and these diffs showed up. Might have been smart tabs...